index.js

```diff
@@ -10,16 +10,20 @@ function fallback() {
 	return cache;
 }
 
-function getEnvLocale() {
-	var env = process.env;
-	var ret = env.LC_ALL || env.LANGUAGE || env.LANG || env.LC_MESSAGES;
+function getEnvLocale(env) {
+	env = env || process.env;
+	var ret = env.LC_ALL || env.LC_MESSAGES || env.LANG || env.LANGUAGE;
 	cache = getLocale(ret);
 	return ret;
 }
 
 function parseLocale(x) {
-	var res = /(?:LC_ALL|LANG|LC_MESSAGES|LC_CTYPE|)="([^"]{2,})"/.exec(x);
```

Comment:
I think most users won't expect it to be used IMHO, since [`LL_CTYPE`](http://www.gnu.org/software/libc/manual/html_node/Locale-Categories.html#Locale-Categories) is used to determine how to test whether characters are letters, digits, punctuation, and interpretation of multi-byte chars.